hamlet/celery.py

```diff
@@ -277,3 +278,97 @@ def workflow_export(
             raise e
     finally:
         os.unlink(out_f_name)
+
+
+# Creates a JSON file which details all the image URLs in a Subject Set, along
+# with that image's associated Subject information. JSON will be of the format:
+#
+# [ [ "http://image.url/example.png",
+#     "{ \"subject_info\": \"is a stringified JSON object\" }" ],
+#   ... ]
+#
+# To be used with the Zooniverse ML Subject Assistant web app (https://github.com/zooniverse/zoo-ml-subject-assistant),
+# and the associated (and currently unnamed) Microsoft machine learning service.
+# ------------------------------------------------------------------------------
+@app.task(bind=True)
+def ml_subject_assistant_export_to_microsoft(
+    self,
+    export_id,
+    access_token,
+):
+    export = MLSubjectAssistantExport.objects.get(pk=export_id)
+    data = []  # Keeps track of all data items that needs to written into a Microsoft-friendly JSON format.
+    
+    # Retrieve all Subjects from a Subject Set
+    try:
+        export.status = MLSubjectAssistantExport.RUNNING
+        export.save()
+        with SocialPanoptes(bearer_token=access_token) as p:
+            subject_set = SubjectSet.find(export.subject_set_id)
+            
+            # Process each Subject
+            for subject in subject_set.subjects:
+                
+                # Create a data item for each image URL in the Subject
+                frame_id = 0
```

Comment:
I'd consider using the `enumerate` function here:

```python
for frame_id, location in enumerate(subject.locations):
```

Reply:
👌